hawkbit-core/src/main/java/org/eclipse/hawkbit/exception/EventHandlerChain.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2021 Bosch.IO GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.exception;
+
+/**
+ * A chain where an event is passed through all handlers until one is found that
```

Comment:
I would rename occurrences of `event` to `error`, since this is not really an event handler.
also in the parameter documentation